front50-api/src/main/java/com/netflix/spinnaker/front50/api/model/Timestamped.java

```diff
@@ -38,7 +35,6 @@ default void setCreatedAt(Long createdAt) {
     // do nothing
   }
 
-  @JsonIgnore
```

Comment:
@JsonIgnore annotations moved to @JsonIgnoreProperties annotation in `TimestampedMixins` class: https://github.com/mochacat/front50/pull/1/files#diff-6af7e4f7bdf38ec5e57797074358e324d5dc7bf59c6f42f1b633ebe1e7570131